assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -42,11 +47,25 @@ class GettingStartedOverview extends React.Component<Props, State> {
     this.setState({color: color.hex});
   };
 
+  async componentWillUnmount() {
+    this.createOrUpdateWidgetConfig();
+  }
+
+  createOrUpdateWidgetConfig = () => {
+    const {color, title, subtitle, account_id} = this.state;
+    API.createOrUpdateWidgetConfig(
+      {color, title, subtitle, account_id},
+      this.state.id
+    )
+      .then((res) => console.log(res))
```

Comment:
happy to leave these in for now for debugging, but maybe add some text to make it more obvious what is being logged (e.g. `console.log('Updated widget config:', res)`, etc)

Reply:
^